lte/gateway/configs/pipelined.yml

```diff
@@ -67,7 +67,7 @@ meter:
   idle_timeout: 0
 
 enforcement:
-  poll_interval: 2
+  poll_interval: 10
```

Comment:
Can you give some context on how you arrived on number 10 in commit log or in config file?

Reply:
That's just a good number I've used in cwf scale tests. I think it's a safe number to merge until we can get a accurate benchmark for different values